### PR TITLE
Add timezones

### PR DESCRIPTION
**Pull request by canyon289**
Fixing timezones on website

image

### CODE REVIEW

**Comment by OriolAbril** [comment]
I would also include the 15 to give a better idea of what is the time constraint (i.e. people in India will probably not really care about the 9 am GMT+2 but about the expected end time to see if they will need to be awake until very late)

**Comment by canyon289** [comment]
Ah good point, Lets double check conference length then before posting. I'll update our proposed google sheet to make sure it all fits together

**Comment by canyon289** [comment]
Dont merge this PR as we are still figuring out timezone nonsense

**Comment by canyon289** [comment]
Once Aki, Viola, and Chris have responded to the email we can set the schedule and get better end estimates

**Comment by OriolAbril** [comment]
I think we can now merge this?

**Comment by canyon289** [comment]
If it looks good please go ahead and do so. Wont be at my computer for a bit

File: _layouts/2017_home.html [hunk 1]
```diff
@@ -16,7 +16,9 @@
                 <div class="intro-heading">PymCon 2020<br></div>
                 <div class="intro-lead-in">An asynchronous-first virtual conference for the Bayesian community</div>
                 <div class="intro-lead-in-sub">October 31st</div>
-                <div class="intro-lead-in-sub">Location: Global</div>
+                <div class="intro-lead-in-sub">Africa/Asia/Europe: 9am GMT+2 </div>
```
Comment (OriolAbril) [inline _layouts/2017_home.html]:
```suggestion
                <div class="intro-lead-in-sub">Africa/Asia/Europe: 9-15 GMT+2 </div>
```

File: _layouts/2017_home.html [hunk 2]
```diff
@@ -16,7 +16,9 @@
...
+                <div class="intro-lead-in-sub">Americas: 9am GMT-7 </div>
```
Comment (OriolAbril) [inline _layouts/2017_home.html]:
```suggestion
                <div class="intro-lead-in-sub">Americas: 9-15 GMT-7 </div>
```

**Review by OriolAbril — APPROVED** [review]

**Pull request merged**